plugins/BEdita/API/src/Model/Action/UpdateAssociatedAction.php

```diff
@@ -14,12 +14,14 @@
 namespace BEdita\API\Model\Action;
 
 use BEdita\Core\Model\Action\BaseAction;
+use Cake\Database\Expression\QueryExpression;
 use Cake\Datasource\Exception\RecordNotFoundException;
 use Cake\ORM\Association;
+use Cake\ORM\Association\BelongsToMany;
 use Cake\Utility\Hash;
 
 /**
- * Command to add links between entities.
+ * Command to update links between entities.
```

Comment:
This action is only a wrapper around concrete implementations of `BEdita\Core\Model\Action\UpdateAssociatedAction`. We introduced in `BEdita\API` in #926  to parse the `$request` data, fetch all required entities and pass all the pre-populated data structures to the downstream Action. Therefore, I think it fits in BEdita/API, since another invocation of the downstream action wouldn't probably have a Request object ready for use… 🤔 

👍  for improving docblock. I'm on it.